subprojects/emcee/action/build.gradle.kts

```diff
@@ -5,16 +5,15 @@ plugins {
 }
 
 dependencies {
-    api(projects.subprojects.emcee.shared)
-    implementation(projects.subprojects.emcee.queueBackendApi)
+    api(projects.subprojects.emcee.queueClientModels)
+    api(libs.moshiAdapters) {
+        "used because EmceeConfig serialized to JSON for testing. Replace as soon as possible"
```

Comment:
but replaced how? do you have an idea?

Reply:
No. But I want to leave an intention. If I could I would replace